app/code/community/Algolia/Algoliasearch/Helper/Algoliahelper.php

```diff
@@ -109,52 +109,37 @@ public function mergeSettings($index_name, $settings)
         return $onlineSettings;
     }
 
-    public function handleTooBigRecords(&$objects, $index_name)
+    public function addObjects($objects, $indexName)
     {
-        $long_attributes = array('description', 'short_description', 'meta_description', 'content');
+        $currentCET = new DateTime('now', new DateTimeZone('Europe/Paris'));
+        $currentCET = $currentCET->format('Y-m-d H:i:s');
+        
+        $modifiedIds = array();
 
-        $good_size = true;
+        foreach ($objects as $key => $object) {
```

Comment:
To keep it very simple like other method of the file:
https://github.com/algolia/algoliasearch-magento/blob/221492f938275674b3b0f4865a26dc1f8ece2ffd/app/code/community/Algolia/Algoliasearch/Helper/Algoliahelper.php#L34-L76
that are just a bridge where we can do stuff

I tend to prefer seeing:
```
  if ($this->config->isPartialUpdateEnabled()) {
            $index->partialUpdateObjects($objects);
        } else {
            $index->addObjects($objects);
        }
```
very quickly and not at the end of a long function

The method name can be generic like `alterRecords` or `modifyRecords` and inside you can have two different submethods